Dockerfile

```diff
@@ -2,10 +2,13 @@ FROM maven:3.5.2 as builder
 MAINTAINER matt.brewster@base2s.com
 COPY . /build
 WORKDIR /build
-RUN mvn clean package
+RUN mvn versions:set -DnewVersion=1.1.0; \
```

Comment:
hm, I am a little ambivalent about this: It's nice to fix that version here, because then in the future with new versions in the pom, we do not need to update the `Dockerfile`. On the other hand, `1.1.0` is an actual version of that plugin, so it feels kind of misleading to use this here. What about setting the version to something fixed like `fixed` or `docker` - so the intent becomes a little clearer?

Reply:
Just fix the rest of the dockerfile - it’s essentially just search and replace 1.1.0